### ISSUE TITLE
How to normalize a tensor to [-1,1] in tensorflow??? 

### ISSUE THREAD

**Issue by longmalongma** [krrish94/nerf-pytorch#16]
How to normalize a tensor to [-1,1] in tensorflow???
